<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportTab</class>
 <widget class="QWidget" name="ImportTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>521</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="tabTableLayout"/>
  </widget>
  <widget class="QPushButton" name="importButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Select File</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>351</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="dataSelectionLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Plot 1: Select Data to Visualize</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="FileSelectionComboBox">
      <property name="placeholderText">
       <string>Select File For Visualization</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>X Data (select one):</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="xDataListWidget"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Y Data (multiple selection possible):</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="yDataListWidget"/>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxSeperate">
      <property name="text">
       <string>Seperate Plots</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PlotButton">
      <property name="text">
       <string>Plot</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="NewPlotButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Another One</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
